docs/en/reference/api/python-api-overview.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: API Overview
+pagename: python-api-overview
+lang: en
+---
+
+# API Overview
+
+**Note:** For more information about the {% include product %} API, see our [API documentation](http://developer.shotgridsoftware.com/python-api/) .
+
+The {% include product %} Application Programming Interface (API) enables users to easily integrate their tools with {% include product %}. Through it you can create automated processes, integrate with many third party software packages, and communicate with existing tools within your studio. Because the needs of every studio are very different, the {% include product %} API provides powerful, low-level functionality and leaves most of the business logic to you.
```

Comment:
I suggest `The {% include product %} Python Application Programming Interface` because we also have REST API

Reply:
Fixed, thanks @000paradox000 !